src/components/utils/directory-content-utils.ts

```diff
@@ -74,14 +74,15 @@ export const getColumnsDefinition = (childrenMetadata: Record<UUID, ElementAttri
         headerCheckboxSelection: true,
         checkboxSelection: true,
         flex: 5,
```

Comment:
then remove flex if it's incompatible with pinned column ?
Or comment it with a good commont to not reintroduce it and explain the minWidth as well